src/main/java/seedu/address/model/person/DriverNameContainsStringPredicate.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.model.person;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Driver}'s {@code Name} matches the String given.
+ */
+public class DriverNameContainsStringPredicate implements Predicate<Driver> {
+    private final String keyString;
+
+    public DriverNameContainsStringPredicate(String keyString) {
```

Comment:
DriverNameContainsStringPredicate .java is exactly same as CustomerNameContainsStringPredicate.java.

Maybe can combine them into a class. Use their parent class Person as parameter instead since Customer and Driver extends Person.